regression/test.pl

```diff
@@ -211,13 +216,13 @@ ($$$$)
   -c CMD     run tests on CMD - required option
   -i <regex> options in test.desc matching the specified perl regex are ignored
   -j <num>   run <num> tests in parallel (requires Thread::Pool::Simple)
+  -d         dry-run: print the tests that would be run, but don't actually run them
```

Comment:
A matter of taste: use `-n` instead of `-d`? I think that's more common for `-d` is mostly used for "debug".

Reply:
Ahh, like `make -n`, good point. Done.